src/manual.md

```diff
@@ -1,6 +1,58 @@
 # Deno Manual
 
-[toc]
+## Table of Contents
+
+- [Disclaimer](#disclaimer)
+- [Introduction](#introduction)
+  - [Philosophy](#philosophy)
+  - [Goals](#goals)
+  - [Non-goals](#non-goals)
+- [Setup](#setup)
+  - [Binary Install](#binary-install)
+  - [Build from source](#build-from-source)
+    - [Prerequisites](#prerequisites)
+    - [Other useful commands](#other-useful-commands)
+- [API reference](#api-reference)
+  - [deno types](#deno-types)
+  - [Reference websites](#reference-websites)
+- [Examples](#examples)
+  - [An implementation of the unix "cat" program](#an-implementation-of-the-unix-cat-program)
+  - [TCP echo server](#tcp-echo-server)
+  - [Inspecting and revoking permissions](#inspecting-and-revoking-permissions)
+  - [File server](#file-server)
+  - [Reload specific modules](#reload-specific-modules)
+  - [Permissions whitelist](#permissions-whitelist)
+  - [Run subprocess](#run-subprocess)
+  - [Linking to third party code](#linking-to-third-party-code)
+  - [Using external type definitions](#using-external-type-definitions)
+  - [Testing if current file is the main program](#testing-if-current-file-is-the-main-program)
+- [Command line interface](#command-line-interface)
+  - [Flags](#flags)
+  - [Environmental variables](#environmental-variables)
+  - [Shell completion](#shell-completion)
+  - [V8 flags](#v8-flags)
+  - [Bundling](#bundling)
+  - [Installing executable scripts](#installing-executable-scripts)
+- [Proxies](#proxies)
+- [Import maps](#import-maps)
+- [Program lifecycle](#program-lifecycle)
+- [Internal details](#internal-details)
+  - [Deno and Linux analogy](#deno-and-linux-analogy)
+    - [Resources](#resources)
+    - [Metrics](#metrics)
+  - [Schematic diagram](#schematic-diagram)
+  - [Profiling](#profiling)
+  - [Debugging with LLDB](#debugging-with-lldb)
+  - [Deno Core](#deno-core)
+  - [Updating prebuilt binaries](#updating-prebuilt-binaries)
+  - [Continuous Benchmarks](#continuous-benchmarks)
+  - [Logos](#logos)
+- [Contributing](#contributing)
+  - [Submitting a pull request](#submitting-a-pull-request)
+  - [Changes to `third_party`](#changes-to-third_party)
+  - [Adding Ops (aka bindings)](#adding-ops-aka-bindings)
+  - [Documenting APIs](#documenting-apis)
+    - [Utilize JSDoc](#utilize-jsdoc)
```

Comment:
I’d rather this done at runtime

Reply:
Fixed it. I have added the [remark-toc](https://github.com/remarkjs/remark-toc) plugin.

It'll generate the TOC at runtime